### PR TITLE
I18n own package

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Review by tiberiuichim — APPROVED** [review]
I like this approach.

**Comment by sneridagh** [comment]
Projects can't work on "local path" Volto deps... we need to make a release before it can work.

File: docs/source/upgrade-guide/index.md [hunk 1]
```diff
@@ -15,6 +15,44 @@ This upgrade guide lists all breaking changes in Volto and explains the
...
+-    "i18n": "NODE_ENV=production node src/i18n.js"
++    "i18n": "rm -rf build/messages && NODE_ENV=production i18n --addon"
++  },
+```
+
+and add this to the `dependencies` list:
+
+```diff
++  "dependencies": {
++    "@plone/scripts": "latest"
```
Comment (tiberiuichim) [inline docs/source/upgrade-guide/index.md:42]: maybe it should be "@plone/scripts": "*" ?

Comment (sneridagh) [reply docs/source/upgrade-guide/index.md:42]: I was hesitant... even tempted to pin it. But then we might want an updater script...

Comment (sneridagh) [reply docs/source/upgrade-guide/index.md:42]: Is there a way to mark "whatever one you have already inherited from the environment"? Yesterday trying, * it just got the latest one... :(

Comment (tiberiuichim) [reply docs/source/upgrade-guide/index.md:42]: I think it's not an issue if you get the latest one, but I'm thinking that maybe someone, at one day, will want to pin it at their project level.

**Review by tisto — APPROVED** [review]
LGTM

**Pull request merged**